### DISCUSSION TITLE
asset_base_url bug

### DISCUSSION THREAD

**Discussion by ecdesa** [livewire/livewire#3098] [7. Bugs (archived)]
### Description
In Livewire website under the `Installation` page and `Configuring The Asset Base URL` section you mention `asset_base_url` as the config key for changing base url for livewire assets but in the config file that gets generated the key is `asset_url`.
My problem is neither one is working 😅.

**Context**
- Livewire version: 2.3.5
- Laravel version: 8.12

It might be useful to mention that my OS is windows since most of pathing issues that occur for me is because of fucking windows.

**Comment by ItsMalikJones** [comment]
When you inspect the view, where is it trying to load the assets from?

If you're working locally, you shouldn't have to config the asset url. Just comment that part of the config out and Livewire will use its default behavior.

**Comment by ecdesa** [comment]
@ItsMalikJones
The thing is, I do want to change the default path.

**Comment by austenc** [comment]
@mohamadnorouzi20 What is the error you are getting, _exactly_? What is the path it is trying to load the assets from?

What are you using to serve your Laravel application in your development environment? You mentioned Windows, but could you be more specific?

**Comment by ecdesa** [comment]
@austenc I'm trying to load it from /assets/vendor/livewire/livewire.js but it keeps loading from /vendor/livewire/livewire.js

**Comment by austenc** [comment]
@mohamadnorouzi20  and what is in your config file? Please provide as much details about your setup as you can.
